tests/test_sanity.py

```diff
@@ -973,3 +973,15 @@ def prep():
           if not side_module:
             assert os.path.exists('a.out.js')
             self.assertContained('hello, world!', run_js('a.out.js'))
+
+  def test_embuilder(self):
+    restore_and_set_up()
```

Comment:
How about just skip this test if `is_wasm_backend()`?

Reply:
Ok. I'll rename the test too, then, to `test_embuilder_wasm_backend.`